x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -312,8 +322,8 @@ export class HeadlessChromiumDriver {
     args,
     timeout,
   }: {
-    fn: EvaluateFn;
-    args: SerializableOrJSHandle[];
+    fn: EvaluateFunc<any>;
```

Comment:
Can we use something other than `any`?

Reply:
I don't think so, at least `unknown` does not work.